action_server/src/sema4ai/action_server/_session.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import os
+from typing import TYPE_CHECKING, Optional
+
+import requests
+from pydantic import BaseModel, Field
+
+if TYPE_CHECKING:
+    from sema4ai.action_server._rcc import Rcc
+
+
+log = logging.getLogger(__name__)
+
+
+class RccCertificateSettings(BaseModel):
+    verify_ssl: Optional[bool] = Field(default=None, alias="verify-ssl")
```

Comment:
Is there yet a way to set the legacy_renegotiation via the options?

Reply:
Added support for this. The session context has the option set if the underlying SSL library supports it. There is a log line if it's not supported.

We cannot propagate this to the actions though as there is no environment variable for this.